linalg/densemat.cpp

```diff
@@ -2093,11 +2093,11 @@ void CalcAdjugate(const DenseMatrix &a, DenseMatrix &adja)
 #ifdef MFEM_DEBUG
    if (a.Width() > a.Height() || a.Width() < 1 || a.Height() > 3)
    {
-      mfem_error("CalcAdjugate(...)");
+      mfem_error("CalcAdjugate(...): dimension mismatch");
    }
    if (a.Width() != adja.Height() || a.Height() != adja.Width())
    {
-      mfem_error("CalcAdjugate(...)");
+      mfem_error("CalcAdjugate(...): dimension mismatch");
```

Comment:
We should distinguish these messages.  Perhaps simply by adding integers similar to lines 1679 and 1709.

Reply:
It looks like the first one checks that both matrices are between 1x1 and 3x3, whereas the second checks that they are the same size and symmetric so I'll change the first error message to "unsupported dimensions".